packages/winston/tests/winston.spec.ts

```
@@ -0,0 +1,179 @@
+import { Writable } from 'stream';
+import { describe, expect, it } from 'vitest';
+import * as winston from 'winston';
+import { createLoggerFromWinston } from '../src';
+
+describe('Winston', () => {
+  let log: string = '';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)